.ci/acceptance-tests/inspec-post-merge.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Service account credentials for GCP to allow terraform to work
+export GOOGLE_CLOUD_KEYFILE_JSON="/tmp/google-account.json"
+export GOOGLE_APPLICATION_CREDENTIALS="/tmp/google-account.json"
+# Setup GOPATH
+export GOPATH=${PWD}/go
+
+# CI sets the contents of our json account secret in our environment; dump it
+# to disk for use in tests.
+set +x
+echo "${TERRAFORM_KEY}" > /tmp/google-account.json
+export GCP_PROJECT_NUMBER=${PROJECT_NUMBER}
+export GCP_PROJECT_ID=${PROJECT_NAME}
+export GCP_PROJECT_NAME=${PROJECT_NAME}
+set -x
+
+gcloud auth activate-service-account terraform@graphite-test-sam-chef.iam.gserviceaccount.com --key-file=$GOOGLE_CLOUD_KEYFILE_JSON
+# TODO(slevenick): Check to see if we have already run this
```

Comment:
I'm totally down with the "get it working now, improve it as we go" model here.  Is that a priority for you, or more of an eventually-TODO?  I feel like a pass rate of 90+% or so should be a requirement for the inspec tests blocking PR generation, and I'm not sure if you'll hit that or not as-is.

Reply:
It's more of a I'll do this next, and this is where that code would go. Definitely plan on doing this before blocking any generation builds on the inspec-test step

Reply:
Awesome, LGTM.